UI/src/router.js

```diff
@@ -50,20 +50,30 @@
       linkProperty: function (linkId) {
         applicationModel.selectLayer('linkProperty');
         backend.getRoadLinkByLinkId(linkId, function (response) {
-          if (response.source == 1) {
+          if (response.success) {
             eventbus.once('linkProperties:available', function () {
               models.selectedLinkProperty.open(response.id);
             });
-          } else if (response.source == 2) {
-            eventbus.once('linkProperties:available', function () {
-              eventbus.trigger('roadLinkComplementaryCheckBox:check');
-              eventbus.trigger('roadLinkComplementary:show');
+
+            if (response.source === 1) {
               eventbus.once('linkProperties:available', function () {
                 models.selectedLinkProperty.open(response.id);
               });
-            });
+            } else if (response.source === 2) {
+              eventbus.once('linkProperties:available', function () {
+                eventbus.trigger('roadLinkComplementaryCheckBox:check');
+                eventbus.trigger('roadLinkComplementary:show');
+                eventbus.once('linkProperties:available', function () {
+                  models.selectedLinkProperty.open(response.id);
+                });
+              });
+            }
+            map.setCenter(new OpenLayers.LonLat(response.middlePoint.x, response.middlePoint.y), 12);
+          }
+          else
+          {
+            //might be nice to show error message for user if roadlink  applied to #linkProperty/ url does not exist
```

Comment:
I agree with the idea of the show error message for user if roadlink doesn't exist.
If we don't have the message yet, maybe it's better make this comment like a TODO.